pkgs/development/beam-modules/build-mix.nix

```diff
@@ -61,7 +61,7 @@ let
       runHook preConfigure
 
       ${./mix-configure-hook.sh}
-      ${lib.optionalString (!isNull appConfigPath)
```

Comment:
Especially expression like this one shouldn't go through review as they complicate the code a lot.